doc/matlab.en.tm

```diff
@@ -1,11 +1,21 @@
-<TeXmacs|1.0.7.11>
+<TeXmacs|1.99.18>
 
-<style|tmdoc>
+<style|<tuple|tmdoc|old-spacing|old-dots|old-lengths>>
 
 <\body>
   <tmdoc-title|The <name|Matlab> plug-in>
 
-  <tmdoc-copyright|2011|Joris van der Hoeven>
+  <\traverse>
+    <\branch>
+      Short description
+    </branch|matlab-abstract.en.tm>
+
+    <branch|Installation|matlab-install.en.tm>
+
+    <branch|Example session|matlab-demo.en.tm>
+  </traverse>
+
+  <tmdoc-copyright|2021|Tilda <abbr|A.> Steiner>
```

Comment:
It would be better to reserve Joris's name. And the copyright year should be `2011-2021` which indicates the history of the doc.